Polyhedron/include/CGAL/Polyhedron_3_to_lcc.h

```diff
@@ -110,10 +118,18 @@ namespace CGAL {
     }
     CGAL::Polyhedron_3<typename LCC::Traits> P;
     ais >> P;
-    return import_from_polyhedron_3<LCC, CGAL::Polyhedron_3
+    return polyhedron_3_to_lcc<LCC, CGAL::Polyhedron_3
                                     <typename LCC::Traits> > (alcc, P);
   }
 
+template < class LCC >
+[[deprecated("Use polyhedron_3_flux_to_lcc instead")]]
+typename LCC::Dart_descriptor
+import_from_polyhedron_3_flux(LCC& alcc, std::istream& ais)
+{
+  return polyhedron_3_flux_to_lcc(alcc, ais);
+}
+
```

Comment:
remove this old name; indeed this function is not documented in cgal, thus there is no need for backward compatibility